### ISSUE TITLE
Regional Analysis Settings Confirmation

### ISSUE THREAD

**Issue by trevorgerhardt** [conveyal/analysis-ui#603]
There should be a screen before running a regional analysis showing the exact settings that will be used for that regional analysis. It's not explicitly clear how the settings for single point translate to regional (for example, the "time cutoff" === "max trip duration minutes")

**Comment by ansoncfit** [comment]
With recent changes, the main priority is to allow multiple projects/scenarios to be selected.  I would suggest the following:

Prefix: [text field, defaults to Batch 1 or something similar]
Project/Scenario: [current project][current scenario][ ➕  add]
Opportunity dataset: [multi-select, defaults to current opportunity dataset][ ➕ add]

Analyses to be run:
* Batch 1: [project/scenario] to [opportunity dataset]
* ... all combinations of selected projects/scenarios and opportunity datasets (limit to 20 for now?)

Profile request settings:
...

✔️ Confirm

**Comment by abyrd** [comment]
With workers under testing, we're now in a position to trigger multiple travel time cutoffs and percentiles at once. The change to the request is relatively simple - just add two arrays cutoffsMinutes (as opposed to the current singular maxTripDurationMinutes, which will be set automatically based on the highest cutoff and some decay function parameters) and `percentiles` (the array, as opposed to the currently used field travelTimePercentile. If the backend sees a non-null cutoffsMinutes, it will assume it's a newer worker and use those cutoffs and the percentiles array. 

To constrain the number of different cutoffs and percentiles supplied at once, we discussed providing an array of checkboxes and requiring an individual click to enable each one, a minimal "speed bump" that requires people to consciously choose to add analyses. Higher travel times are especially problematic since doubling the max travel time could increase the searched area by a factor of roughly pi, tripling the computation. We want to directly ask people not to enable those unless they really want them.

Here's a mockup:
![image](https://user-images.githubusercontent.com/112871/73381789-4d5a1900-4301-11ea-951d-773e1515d25e.png)

Basically just three columns of checkboxes, whose values just collated into the arrays cutoffsMinutes and percentiles.

Additionally we are now testing distance decay functions in regional analyses. So it would be good to also have a drop-down to set a field decayFunction to one of {step, linear, sigmoid, exponential} and a field to enter a numeric parameter (which will typically be interpreted as the rolloff slope).

**Comment by abyrd** [comment]
The multiple cutoffs and percentiles could certainly be combined with @ansoncfit 's suggestion to allow multiple scenarios, projects, or opportunity datasets. For the first two, we'd just iterate over the scenarios and fire off one near-identical regional analysis request per scenario. Multiple opportunity datasets are actually handled more like percentiles and cutoffs internally, but are not yet implemented (unioning the destination grids is tricky). So the multiple opportunity datasets can definitely be left out for now, but the multiple scenarios seems like a more straightforward convenience for users.
